.travis.yml

```diff
@@ -2,9 +2,27 @@ language: python
 
 python: 2.7
 
-install: "pip install flake8"
+addons:
+  - firefox: "42.0"
```

Comment:
You should be able to use `latest` or `latest-esr` to avoid this needing to be maintained. See https://docs.travis-ci.com/user/firefox/ for details.